### ISSUE TITLE
Model weights for Segformer/Densedepth / Units for table 4

### ISSUE THREAD

**Issue by gabriel-tjio** [mihirp1998/Diffusion-TTA#5]
Hi Mihir,

would it be possible to share the pretrained model weights for Segformer/DenseDepth?

1) The units for Table 4 are not mentioned in the paper, could you share what was the metric used for (depth estimation and semantic segmentation)?

2) For clean performance, Segformer and DenseDepth are both 66.1 and 92.4.  I wasn't able to get similar scores with the mmsegmentation weights (Segformer b5 640x640 or the pretrained nyu weights for DenseDepth). Would it be possible to shed some light on this? 

Perhaps the pretrained weights (before applying DiffusionTTA and the metrics that were used could help solve this.

Thanks very much for any advice you can give on this. 

![table4_diffusiontta](https://github.com/mihirp1998/Diffusion-TTA/assets/47103879/e08d45f2-671f-4f37-8d74-ceb78dfbbafa)

**Comment by mihirp1998** [comment]
Thanks for pointing this out. We clearly missed mentioning the metrics.

For semantic segmentation we use the mean classification accuracy. Also we subsample the number of test datapoints to 1500 as doing TTA training on all the examples is very compute heavy, therefore the numbers might not be an exact match.

We use the following code for loading segformer from mmseg.

```
from mmseg.apis import init_model
mmseg_config = "mmsegmentation/configs/segformer/segformer_mit-b0_8xb2-160k_ade20k-512x512.py"
checkpoint = "mmsegmentation/segformer_mit-b0_512x512_160k_ade20k_20210726_101530-8ffa8fda.pth"
model = init_model(mmseg_config, checkpoint, device="cpu")

```

I need to double check for depth prediction.

**Comment by gabriel-tjio** [comment]
Thanks Mihir!
Did you find out what the metrics are for depth prediction?

**Comment by gabriel-tjio** [comment]
Do drop a line when you find out the metrics for depth estimation, thanks very much!

**Comment by gabriel-tjio** [comment]
Thanks very much for your help, Mihir!

**Comment by gabriel-tjio** [comment]
Hi Mihir, I was wondering if the authors could provide the subsampling and evaluation code for segmentation and depth estimation  (ADE20k and NYU2k). It would be a great help for facilitating comparisons with your work and may also open up additional research directions. Thanks for your kind attention and hope to hear from you.
